### ISSUE TITLE
does validation set = test set in Section E (supp material)

### ISSUE THREAD

**Issue by MarcTLaw** [facebookresearch/hgnn#4]
Hi, I really enjoyed reading your paper.

I tried to run your code on different datasets and realized that the validation set is the same as the test set for the datasets used in Section E (use_valid_set = False in data_process.py). Since the datasets are relatively small and it corresponds to 10-fold cross validation scores anyway, I don't think it's a problem.
Could you please confirm that the scores are reported when validation = test for Section E? The experiments take some time to run so it would save me some time.

Thanks in advance,
Marc

**Comment by leuchine** [comment]
Hi Marc:

Thanks for your interest in our paper. As these datasets are small, most papers did not split into train, valid and test sets. We tried both settings, use_valid_set = False and use_valid_set = True (to enable  early stop during training). The results did not differ a lot. Please feel free to use either setting (but using early stop seems the right way to do). Thanks!

Best Regards,
Qi
